oak_abi/proto/label.proto

```diff
@@ -65,6 +66,12 @@ message WebAssemblyModuleTag {
   bytes web_assembly_module_hash_sha_256 = 1;
 }
 
+// Policies related to modules, referring to the signature of the native WebAssembly node.
+message WebAssemblyModuleSignatureTag {
+  // Public key counterpart of the private key used to sign WebAssembly module.
+  bytes public_key = 1;
```

Comment:
Please add some more details about the format of such key.

Reply:
Minisign doesn't specify the serialization format. Looks like they just concatenate the hash algorithm, keynum and the key itself:
https://github.com/jedisct1/rust-minisign/blob/1d909fa0272b7177cb86717f280f104de2658fbc/src/public_key.rs#L100-L116

Will add a comment that it's a serialized `minisign` key